src/ldp/representation/RepresentationMetadata.ts

```diff
@@ -159,13 +160,18 @@ export class RepresentationMetadata {
    * @param subject - Subject of quad to add.
    * @param predicate - Predicate of quad to add.
    * @param object - Object of quad to add.
+   * @param graph - Optional graph of quad to add.
    */
   public addQuad(
     subject: NamedNode | BlankNode | string,
     predicate: NamedNode | string,
     object: NamedNode | BlankNode | Literal | string,
+    graph?: MetadataGraph,
```

Comment:
```suggestion
    graph: MetadataGraph = defaultGraph,
```

Reply:
and below as well

Reply:
Changing it here is fine, but changing it in `removeQuad` changes the behaviour. With the current implementation it would remove that metadata from any graph where it was found. Changing to the default parameter would not delete from the generated graph if no parameter was passed. 

The reason I chose the first behaviour was to make sure that it's not necessary to think of the graph unless you really want to.

Reply:
Excellent, leave as-is.